src/pages/category/Category.js

```diff
@@ -1,48 +1,65 @@
-import React from 'react';
+import { useEffect } from 'react';
+import { getCategories } from '../../actions/categoryActions';
 import CategoryCard from '../../components/cards/CategoryCard';
+import Loader from '../../components/loader/Loader';
+import { useCategory } from '../../context/categoryContext';
 
 const Category = () => {
-  const categories = [
-    {
-      id: 1,
-      categoryName: 'movies',
-      image:
-        'https://res.cloudinary.com/svj/image/upload/v1652541674/movies_u35yi1.jpg',
-    },
-    {
-      id: 2,
-      categoryName: 'technology',
-      image:
-        'https://res.cloudinary.com/svj/image/upload/v1652541674/technology_plfcs8.jpg',
-    },
-    {
-      id: 3,
-      categoryName: 'history',
-      image:
-        'https://res.cloudinary.com/svj/image/upload/v1652541674/history_pstz1r.jpg',
-    },
-    {
-      id: 4,
-      categoryName: 'programming',
-      image:
-        'https://res.cloudinary.com/svj/image/upload/v1652541674/programming_zjtsdg.jpg',
-    },
-  ];
+  // const categories = [
+  //   {
+  //     id: 1,
+  //     categoryName: 'movies',
+  //     image:
+  //       'https://res.cloudinary.com/svj/image/upload/v1652541674/movies_u35yi1.jpg',
+  //   },
+  //   {
+  //     id: 2,
+  //     categoryName: 'technology',
+  //     image:
+  //       'https://res.cloudinary.com/svj/image/upload/v1652541674/technology_plfcs8.jpg',
+  //   },
+  //   {
+  //     id: 3,
+  //     categoryName: 'history',
+  //     image:
+  //       'https://res.cloudinary.com/svj/image/upload/v1652541674/history_pstz1r.jpg',
```

Comment:
you can remove this code, if no longer needed. Or can mention - "TODO" before it.